bin/n

```diff
@@ -793,15 +797,15 @@ install() {
   log installing "${g_mirror_folder_name}-v$version"
 
   local url="$(tarball_url "$version")"
-  is_ok "${url}" || abort "download preflight failed for '$version' (${url})"
+  is_ok "${url}" || abort "download preflight failed for '$version' '$(display_masked_url ${url})'"
```

Comment:
The password may have a space in it, so the url needs to be quoted to pass the expansion as a single argument:
```suggestion
  is_ok "${url}" || abort "download preflight failed for '$version' '$(display_masked_url "${url})"'"
```

(Applies to other uses too.)